src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -374,6 +425,12 @@ const ChatInfoScreen = () => {
           }}
         />
       )}
+      <ModalChangeName
+        onCloseModal={closeOnNameChange}
+        isOpen={isOpenModalChangeName}
+        name={channelInfo?.rawJson?.channel?.is_name_custom ? channelName : ''}
+        onSave={onSaveChangeName}
+      />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `ModalChangeName` component seems to be well implemented. Just make sure that all the props passed to this component are properly used and handled.

```diff
+  <ModalChangeName
+    onCloseModal={closeOnNameChange}
+    isOpen={isOpenModalChangeName}
+    name={channelInfo?.rawJson?.channel?.is_name_custom ? channelName : ''}
+    onSave={onSaveChangeName}
+  />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->